glwf/v3.1 fails to build on FreeBSD (missing -lXinerama)

On FreeBSD 10.1-STABLE, to get `go get github.com/go-gl/glfw/v3.1/glfw` to build correctly I needed to add `-lXinerama` to the end of the [`#cgo freebsd`… lines in `build.go`](https://github.com/go-gl/glfw/blob/master/v3.1/glfw/build.go#L49:L50).

Without that the error was:

```
# github.com/go-gl/glfw/v3.1/glfw
/tmp/go-build819051931/github.com/go-gl/glfw/v3.1/glfw/_obj/glfw_x11_init.o: In function `initExtensions':
./glfw/src/x11_init.c:530: undefined reference to `XineramaQueryExtension'
./glfw/src/x11_init.c:534: undefined reference to `XineramaIsActive'
/tmp/go-build819051931/github.com/go-gl/glfw/v3.1/glfw/_obj/glfw_x11_monitor.o: In function `_glfwPlatformGetMonitors':
./glfw/src/x11_monitor.c:218: undefined reference to `XineramaQueryScreens'
collect2: error: ld returned 1 exit status
```

--- a/v3.1/glfw/build.go
+++ b/v3.1/glfw/build.go
@@ -46,7 +46,7 @@ package glfw
 #cgo freebsd,wayland CFLAGS: -D_GLFW_WAYLAND -D_GLFW_EGL -D_GLFW_HAS_DLOPEN
 
 // Linker Options:
-#cgo freebsd,!wayland LDFLAGS: -lGL -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm
-#cgo freebsd,wayland LDFLAGS: -lGL -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm
+#cgo freebsd,!wayland LDFLAGS: -lGL -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm -lXinerama
+#cgo freebsd,wayland LDFLAGS: -lGL -lX11 -lXrandr -lXxf86vm -lXi -lXcursor -lm -lXinerama
 */
 import "C"

